app/services/search/user.rb

```diff
@@ -25,7 +25,8 @@ def prepare_doc(hit)
           "comments_count" => source["comments_count"],
           "badge_achievements_count" => source["badge_achievements_count"],
           "last_comment_at" => source["last_comment_at"],
-          "roles" => source["roles"]
+          "roles" => source["roles"],
+          "user_id" => source["id"]
```

Comment:
This was a minor bug fix, I noticed when inspecting the frontend that users did not have data-content-user-id set bc it was calling `user_id` on the Elasticsearch hash which was missing for the user so I added it. 
```
data-content-user-id="'+article.user_id+'
```